game.js

```diff
@@ -50,14 +50,15 @@ const inter = setInterval(() => {
     timeLeft--
 }, 1000);
 
-setTimeout(() => {
+let intervalId = setTimeout(() => {
```

Comment:
This name a little confusing, the variable holds a timeout rather than an interval! Semantics!